connectors/yarpc/yarpc.go

```diff
@@ -617,7 +617,8 @@ func (c *Connector) CheckSchemaStatus(ctx context.Context, scope, namePrefix str
 // CreateScope creates the scope specified
 func (c *Connector) CreateScope(ctx context.Context, scope string) error {
 	request := &dosarpc.CreateScopeRequest{
-		Name: &scope,
+		Name:      &scope,
+		Requester: dosa.GetUsername(),
```

Comment:
Where do we store the requester information?

Reply:
In DOSA! The gateway will store all scope metadata. The owning user is the only thing the server cannot compute so it must be provided by the client. If anyone can think of any other attributes it would be useful for the client to sent, let me know!